### DISCUSSION TITLE
Signin works locally with "npm run dev", but not works when hosted

### DISCUSSION THREAD

**Discussion by Solemensis** [supabase/supabase#9644] [Questions]
When i host my website on netlify, signIn gives 404 error instead working like it did on local.

This is console error:
![image](https://user-images.githubusercontent.com/96355888/196378311-21cdd5d7-7c16-4ae3-bb15-213d2a513313.png)

PS: I'm using nuxt supabase pack with nuxt

**Comment by Sukikiroi** [comment]
Hello, can you show us your Signin Function plz

**Reply by Solemensis** [reply]
![image](https://user-images.githubusercontent.com/96355888/196380425-876ea923-56cb-407d-8fe8-c88a2ce1ca72.png)


this works locally again btw

**Comment by GaryAustin1** [comment]
Console log your supabase client object and see if the supabaseKey and supabaseUrl are getting set to what you expect.

**Reply by Solemensis** [reply]
> Console log your supabase client object and see if the supabaseKey and supabaseUrl are getting set to what you expect.

they are what they needed to be in both local host, and netlify host. but only local host works.

**Comment by MikeBman** [comment]
Did you make any progress on this? I'm having a similar problem just getting the example Nuxt quickstart sign in process to work beyond dev mode. For me, it never works after building a static site, even on localhost.
